pom.xml

```diff
@@ -18,7 +18,7 @@
    <properties>
            <jmockit.version>1.49</jmockit.version>
            <argLine>-javaagent:${user.home}/.m2/repository/org/jmockit/jmockit/${jmockit.version}/jmockit-${jmockit.version}.jar</argLine>
-           <sonar.organization>usdot-jpo-ode-1</sonar.organization>
+           <sonar.organization>usdot-jpo-ode</sonar.organization>
```

Comment:
why do we need this defined in two places. It seems like there is some redundancy in this CI